src/components/Form/index.js

```diff
@@ -0,0 +1,94 @@
+import { Text, TextInput, TouchableOpacity, View } from "react-native";
+import styles from './styles'
+import { useState } from "react";
+
+
+export default function Form({ onAddNewWord }) {
+
+    const [title, setTitle] = useState('')
+    const [description, setDescription] = useState('')
+
+    const [errorMessage, setErrorMessage] = useState([])
+
+
+    const handleSaveButton = () => {
+        const messageValidate = []
+
+        if (title === '') {
```

Comment:
if (title) should be enough for this.

Reply:
okay will do that and push it again. Since, title goes in the textfield I just want to make sure that the form is not empty. But I will change.